app/views/users/quick_new.html.erb

```diff
@@ -4,4 +4,4 @@
   <%= render partial: 'short_form' %>
 <% else %>
   <p><%= @message %></p>
-<% end %>
+<% end %>
```

Comment:
This is a "meaningless" change (just removed the new line at the end), we should revert it.